README.md

```diff
@@ -29,24 +31,27 @@ This project should demonstrate your ability to:
 
 Before you start writing _any_ code:
 
-- Review the requirements with your partner
-- Create a Trello board
-  - Organize your tasks/stories
-  - Discuss who will do which tasks and when you will target completion
+- High five your team
+- Discuss communication style/feedback style with your team
+- Review the requirements with your team
+- If there are any instructions, follow any instructions that your instructors may give you directions on joining a specific Trello team/organization
```

Comment:
Line 37 is a little awkward - what about:

Follow any instructions given to the class regarding joining a specific Trello team/organization